tests/framework/console/FakeController.php

```diff
@@ -84,6 +84,10 @@ public function actionAksi6()
         return $this->testArray;
     }
 
+    public function actionAksi7(self $typedArgument, $simpleArgument) {
```

Comment:
Please have a look at the rest of `FakeController`. It contains action that are numerically named for some reason. If you don't like it you should fix the whole controller not just 1 random function.